empyrical/stats.py

```diff
@@ -1004,6 +990,208 @@ def cagr(returns, period=DAILY, annualization=None):
 
     return ending_value ** (1. / no_years) - 1
 
+def capture(returns, factor_returns, period=DAILY):
+    """
```

Comment:
What kind of reference ? 

- http://www.morningstar.com/InvGlossary/upside-downside-capture-ratio.aspx

- http://www.investopedia.com/terms/u/up-market-capture-ratio.asp

Something like that? or am I misunderstanding what you mean by reference ?